packages/site/src/components/TokensView.js

```diff
@@ -1,4 +1,4 @@
-import { Route, Switch, useRouteMatch } from 'react-router-dom';
+import { Route, Routes, useMatch } from 'react-router-dom';
```

Comment:
`useRouteMatch` was deprecated in favor of `useMatch()` in the newer version of React.